pkg/backup/item_collector.go

```diff
@@ -99,6 +100,65 @@ func (r *itemCollector) getGroupItems(log logrus.FieldLogger, group *metav1.APIR
 	return items, nil
 }
 
+// sort items by the names specified in "order".  Items are not in order will be put at the end in original order.
```

Comment:
Since you need to rebase anyway, I wanted to ask you to start the documentation for every function with the name of the function. Example, `// sortResourcesByOrder sorts items by...` I think this is mostly a Go convention.

Reply:
Fixed.